src/main/kotlin/no/nav/familie/ef/iverksett/arena/VedtakhendelseMapper.kt

```diff
@@ -0,0 +1,35 @@
+package no.nav.familie.ef.iverksett.arena
+
+import no.nav.familie.ef.iverksett.iverksetting.domene.Iverksett
+import no.nav.melding.virksomhet.vedtakhendelser.v1.vedtakhendelser.WSApplikasjoner
+import no.nav.melding.virksomhet.vedtakhendelser.v1.vedtakhendelser.WSAvslutningsstatuser
+import no.nav.melding.virksomhet.vedtakhendelser.v1.vedtakhendelser.WSBehandlingstema
+import no.nav.melding.virksomhet.vedtakhendelser.v1.vedtakhendelser.WSVedtakHendelser
+import java.text.SimpleDateFormat
+import java.util.*
+import javax.xml.datatype.DatatypeFactory
+
+
+fun mapIverksettTilVedtakHendelser(iverksett: Iverksett): WSVedtakHendelser {
+    val wsVedtakHendelser = WSVedtakHendelser()
+
+    val wsAvslutningsstatuser = WSAvslutningsstatuser()
+    wsAvslutningsstatuser.kodeRef = iverksett.vedtak.vedtaksresultat.name
+    wsVedtakHendelser.avslutningsstatus = wsAvslutningsstatuser //Arena støtter ikke avslått?
+
+    val wsBehandlingstema = WSBehandlingstema()
+    wsBehandlingstema.value = "ab0272" //Enslig forsørger
+    wsVedtakHendelser.behandlingstema = wsBehandlingstema
+
+    val wsApplikasjoner = WSApplikasjoner()
+    wsApplikasjoner.value = "EF"
+    wsVedtakHendelser.hendelsesprodusentREF = wsApplikasjoner
+
+    wsVedtakHendelser.aktoerID = iverksett.søker.personIdent
+    wsVedtakHendelser.applikasjonSakREF = iverksett.fagsak.eksternId.toString()
+    wsVedtakHendelser.hendelsesTidspunkt = DatatypeFactory.newInstance().newXMLGregorianCalendar(
```

Comment:
`DatatypeFactory.newInstance()` koster en del hver gang for å initiere (skal være thread safe)
Vi hadde kunnet vurdert å bruke en binding.xml for tjenestespesifikasjoner sånn att vi kunde brukt LocalDateTime :P

Reply:
Ah, det visste jeg ikke. Har løst dette ved å lage eget objekt i stedet for å være avhengig av generering fra xsd.